cmd/pulumi-test-language/interface.go

```diff
@@ -274,6 +275,20 @@ type compiledReplacement struct {
 	Replacement string
 }
 
+// programOverrides represent overrides whereby a test may specify a set of hardcoded or pre-generated programs to be
+// used, in place of running GenerateProject on source PCL. This is useful for testing SDK functionality when the
+// requisite program code generation is not yet complete enough to support generating programs which exercise that
+// functionality.
+type programOverride struct {
+	// The name of the test to which this override applies.
+	Test string
```

Comment:
We don't need this here? It's already the map key for ProgramOverrides in the test token.

Reply:
Ack; fixed :+1: